R/utils.R

```diff
@@ -0,0 +1,325 @@
+# Helper functions for do_read_bib()
+
+#' Parse (at)string keys
+#'
+#' Returns a table with all the string values and their value to be used on the
+#' `bib` file.
+#'
```

Comment:
Could we add an `@param` describing input? This is for long-term maintenance benefits as the function is internal.

Reply:
I added documentation of params and also some examples on each internal function, hopefully that would help maintaining them

Reply:
I added the examples also as tests on `test-utils.R`